### PR TITLE
Add how to upgrade while keeping current deployment

### PR DESCRIPTION
**Pull request by returntrip**
@jlebon FYI, incorporated your feedback and created PR. Thanks.

### CODE REVIEW

File: modules/ROOT/pages/faq.adoc [hunk 1]
```diff
@@ -103,3 +103,22 @@ If, for whatever reason, you need to downgrade the kernel, you can do so by foll
 
 . Reboot the system to apply the changes.
 
+How can I upgrade my system to the next major version (for instance: rawhide or an upcoming Fedora release branch), while keeping my current deployment?::
+
+OSTree allows you to pin deployments (pinning ensures that your deployment of choice is kept and not discarded).
+
+. Assuming that you want to keep your default deployment, issue the following command:
+
+ $ sudo ostree admin pin 0`
```
Comment (miabbott) [inline modules/ROOT/pages/faq.adoc]: Looks like a spurious trailing backtick?

**Comment by miabbott** [comment]
One little nit, otherwise LGTM

File: modules/ROOT/pages/faq.adoc (hunk 1 above)
Comment (returntrip) [reply modules/ROOT/pages/faq.adoc]: Good catch. Fixed. Please review now.

**Comment by miabbott** [comment]
Looks good, thanks!

**Pull request merged**